app/models/github_fetcher/null_issue_response.rb

```diff
@@ -0,0 +1,15 @@
+module GithubFetcher
+  class NullIssueResponse
```

Comment:
ditto. this can all live in Issues. if there are no issues, set `response` to an empty array.